flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js

```diff
@@ -283,7 +283,7 @@ class NavSystem extends BaseInstrument {
         return null;
     }
     onInteractionEvent(_args) {
-        if (this.isElectricityAvailable() || SimVar.GetSimVarValue("L:DCPowerAvailable", "Bool") == 1) {
```

Comment:
It's unclear to me where `isElectricityAvailable` is defined. I'm also not sure if this `A32NX_NavSystem.js` code ever runs or is actively used. If anyone is able to shed some light on this that would be great.

Reply:
`isElectricityAvailable` maybe in the Asobo BaseInstrument class? Our instruments inherit `BaseAirliners` which in turn inherits `NavSystem`, which in turn inherits `BaseInstrument`.